### ISSUE TITLE
B ANOVAs: Proposal to remove columns from results

### ISSUE THREAD

**Issue by jonathon-love** [jasp-stats/jasp-desktop#633]
Hello, @EJWagenmakers, @mmarsman1, @raviselker 

I'm proposing we drop the columns: P(M), P(M|data) and BFM from the B ANOVAs and Linear Regression.

1. They're not useful
-------------------

I understand how these values are calculated, but I do not think they are interesting. There may be some pedagogical exercise in looking at the way P(M) is modified to produce P(M|data), and then the way these are combined to provide BFM; but I think this is the limit of their usefulness, and in practice only the BF10 / BF01 column is interesting (and there's nothing that you can infer from BFM, that can't be more easily infered from BF10)

2. They can be deceptive
--------------

Further, they often produce deceptive results, for example in the following, at a glance it would appear that there is support for the full model:

![screen shot 2015-04-22 at 4 28 00 pm](https://cloud.githubusercontent.com/assets/3240247/7276799/a043f18a-e90c-11e4-96d0-4208067f76b1.png)

however, it becomes clear when we mark *fright* and *disgust* as nuisance that this is not the case:

![screen shot 2015-04-22 at 4 30 03 pm](https://cloud.githubusercontent.com/assets/3240247/7276846/d957d874-e90c-11e4-9a32-631b60f87f27.png)

3. Awkward unclear relationship with the BF10, BF01, Log(BF10) button
------------

what do we actually do for this?

4. Simplicity
------------

without these terms, the tables become simpler, and consistent with the t-tests

**Comment by EJWagenmakers** [comment]
Let's move over the BF10 column to the left to emphasize it's importance.
But that's it for now. The conting package produces many of our columns
too, and as main output. If it turns out later that particular output is
not needed so much we can talk about it. But pruning this now is premature.
EJ

**Comment by EJWagenmakers** [comment]
The example about the misleadingness is not really misleading I think; it
depends on the questions you are asking.
EJ

**Comment by jonathon-love** [comment]
yeah, if there's a good reason to have it, then i'm totally on board with keeping it.

**Comment by jonathon-love** [comment]
> But pruning this now is premature.

i think in software, you want to do things the other way around. start minimal, and add more things.

people hate you taking things away (even when no-one uses it... you get this vocal minority complaining). but adding stuff later can often be well received.

**Comment by EJWagenmakers** [comment]
Yeah but I think the BF10 is hard to parse and the other info is useful.
Just wait until you see the pbr paper
EJ

**Comment by jonathon-love** [comment]
i'm a bit too tired to care too much about this one at the moment, but the part of me which is concerned about us potentially digging ourselves into a hole wants me to say something.

PBR paper hasn't illuminated how these columns are useful.

to reiterate, very happy to keep these columns, just no-one has been able to provide a use-case; i.e. a  scenario or research question (reasonable or otherwise) where these would be useful.

providing superfluous columns that no-one uses is bad, blah blah blah .... (i've made a pretty good case in this thread)

i'm a little bewildered by the resistance to this.

and to reiterate - very happy to keep these columns if there is reason to.

**Comment by jonathon-love** [comment]
OK, so we've decided to remove BF(M) from the results.

Soooo, while we're removing columns that aren't useful to users (or at least no-one has been able to make a case for why they are useful to users), why don't we remove the P(M) and P(M|data) columns too?

**Comment by EJWagenmakers** [comment]
No. They will feature prominently in related code that we develop now.
Please keep them in.
E.J.


********************************************
Eric-Jan Wagenmakers
Department of Psychological Methods, room 2.09
University of Amsterdam
Weesperplein 4
1018 XA Amsterdam
The Netherlands

Web: ejwagenmakers.com
Book: bayesmodels.com
Stats: jasp-stats.org
Email: EJ.Wagenmakers@gmail.com
Phone: (+31) 20 525 6420

“Man follows only phantoms.”
Pierre-Simon Laplace, last words
********************************************

**Comment by jonathon-love** [comment]
> No. They will feature prominently in related code that we develop now.

ok, i will remain open minded (but skeptical in the absence of a user-centred justification) that there is some (user-centric) point to these.

**Comment by jonathon-love** [comment]
ok, i discussed this with maarten, and he made a (user-centric) case for keeping BF(M), so now i think we should keep it.
